runtime/src/genesis_utils.rs

```diff
@@ -219,10 +219,20 @@ pub fn create_genesis_config_with_leader_with_mint_keypair(
     }
 }
 
+pub fn activate_all_features_alpenglow(genesis_config: &mut GenesisConfig) {
+    do_activate_all_features::<true>(genesis_config);
+}
+
 pub fn activate_all_features(genesis_config: &mut GenesisConfig) {
+    do_activate_all_features::<false>(genesis_config);
+}
+
+pub fn do_activate_all_features<const IS_ALPENGLOW: bool>(genesis_config: &mut GenesisConfig) {
     // Activate all features at genesis in development mode
     for feature_id in FeatureSet::default().inactive() {
-        activate_feature(genesis_config, *feature_id);
+        if IS_ALPENGLOW || *feature_id != agave_feature_set::alpenglow::id() {
```

Comment:
To confirm, the idea here is keep AG feature disabled by default but to also have a way to enable the gate for testing ?

Reply:
yeah exactly. it will take several dozen PRs before alpenglow is fully functional on master during which all our existing local cluster tests will be broken. 

once functional we can remove this exception to enable the existing tests before we consider rekeying and releasing.

Reply:
Got it, that seems reasonable if we're going to incrementally move AG stuff over